### PR TITLE
fix:Drawer组件添加 footerStyle 控制台报错

### LINKED ISSUE #20978: [4.x] Drawer添加了 footerStyle后，控制台报错
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[Edit on CodeSandbox](https://codesandbox.io/s/eloquent-bouman-21723?fontsize=14&hidenavigation=1&theme=dark)

### Steps to reproduce
给Drawer添加footerStyle属性

### What is expected?
控制台不报错

### What is actually happening?
控制台报错

| Environment | Info |
|---|---|
| antd | 4.0.0-rc.1 |
| React | latest |
| System | win10 |
| Browser | chrome 79 |

### PR DESCRIPTION
**Pull request by XianZhengquan**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
fix #20978

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |     修复 `Drawer`组件添加`footerStyle`属性后控制台报错      |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
写一下 Changelog

**Comment by XianZhengquan** [comment]
在哪里写

**Comment by yoyo837** [comment]
image

**Comment by XianZhengquan** [comment]
加上了

File: components/drawer/index.tsx [hunk 1]
```diff
@@ -247,6 +247,7 @@ class Drawer extends React.Component<DrawerProps & ConfigConsumerProps, IDrawerS
             'drawerStyle',
             'headerStyle',
             'bodyStyle',
+            'footerStyle',
             'title',
```
Comment (yoyo837) [inline components/drawer/index.tsx]: `footer` 和 `locale` 要不也加上？🤔

Comment (yoyo837) [reply components/drawer/index.tsx]: https://github.com/react-component/drawer 那边也不使用

Comment (XianZhengquan) [reply components/drawer/index.tsx]: 我加吗？

File: components/drawer/style/drawer.less [hunk 2]
```diff
@@ -214,7 +214,7 @@
   }
 
   &-body {
-    flex-grow: 1;
+    flex: 1;
```
Comment (yoyo837) [inline components/drawer/style/drawer.less]: 这里恢复 https://github.com/ant-design/ant-design/issues/20978#issuecomment-575558027

File: components/drawer/index.tsx (hunk 1 above)
Comment (yoyo837) [reply components/drawer/index.tsx]: 是

File: components/drawer/style/drawer.less (hunk 2 above)
Comment (XianZhengquan) [reply components/drawer/style/drawer.less]: 是我重新修改然后再 PR吗

Comment (yoyo837) [reply components/drawer/style/drawer.less]: 就在你当前分支修改，push后这里就变了。

**Comment by XianZhengquan** [comment]
改好了

**Comment by yoyo837** [comment]
[贡献指南](https://next.ant.design/docs/react/contributing-cn#%E7%AC%AC%E4%B8%80%E6%AC%A1%E8%B4%A1%E7%8C%AE)

Run `npm test -- -u` to update the jest snapshots and commit these changes as well (if there are any updates).

**Comment by XianZhengquan** [comment]
> [贡献指南](https://next.ant.design/docs/react/contributing-cn#%E7%AC%AC%E4%B8%80%E6%AC%A1%E8%B4%A1%E7%8C%AE)
> 
> Run `npm test -- -u` to update the jest snapshots and commit these changes as well (if there are any updates).

我现在是运行这个，然后再提交嘛

**Comment by yoyo837** [comment]
> > [贡献指南](https://next.ant.design/docs/react/contributing-cn#%E7%AC%AC%E4%B8%80%E6%AC%A1%E8%B4%A1%E7%8C%AE)
> > Run `npm test -- -u` to update the jest snapshots and commit these changes as well (if there are any updates).
> 
> 我现在是运行这个，然后再提交嘛

运行完，把变更push上来。

**Comment by XianZhengquan** [comment]
> > > [贡献指南](https://next.ant.design/docs/react/contributing-cn#%E7%AC%AC%E4%B8%80%E6%AC%A1%E8%B4%A1%E7%8C%AE)
> > > Run `npm test -- -u` to update the jest snapshots and commit these changes as well (if there are any updates).
> > 
> > 
> > 我现在是运行这个，然后再提交嘛
> 
> 运行完，把变更push上来。

好的

**Comment by XianZhengquan** [comment]
终于跑完了，push 了

**Pull request merged**